src/Propulsion.DynamoStore/DynamoStoreSource.fs

```diff
@@ -154,7 +154,7 @@ type DynamoStoreSource
         mode: EventLoadMode,
         // The whitelist of Categories to use
         ?categories,
-        // Predicate to filter Categories to use
+        // Predicate to filter <c>StreamName</c>'s to use
```

Comment:
FYI I know I do this a lot myself but `<c>` is only meaningful in a `/// <summary>`, so maybe backticking them can/should be the convention in `//` comments

(I had lots of `///` on argument lists but Rider and Vs for last 2y corectly flag that they are not valid)

(No need to fix as there's a stack of similar ones)